lib/SPIRV/libSPIRV/SPIRVExtInst.h

```diff
@@ -257,6 +257,7 @@ template <> inline void SPIRVMap<SPIRVDebugExtOpKind, std::string>::init() {
   add(SPIRVDebug::InlinedAt, "DebugInlinedAt");
   add(SPIRVDebug::ImportedEntity, "DebugImportedEntity");
   add(SPIRVDebug::ModuleINTEL, "DebugModuleINTEL");
```

Comment:
What is the difference between DebugModuleINTEL and DebugModule? Is there a spec extension for DebugModuleINTEL? I can also check?

Thanks